pom.xml

```diff
@@ -25,9 +25,10 @@
   </scm>
 
   <properties>
-    <maven.compiler.source>1.6</maven.compiler.source>
-    <maven.compiler.target>1.6</maven.compiler.target>
+    <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
nope, it's not necessary. And in reality you only need Java 7 to use `idm-user-managment-ldaptive`